shub/image/push.py

```diff
@@ -30,17 +31,22 @@
 @click.option("-v", "--verbose", is_flag=True,
               help="stream push logs to console")
 @click.option("-V", "--version", help="release version")
+@click.option("-S", "--skip-tests", help="skip testing image", is_flag=True)
```

Comment:
Let's move this line to the end of the section, to match its order with function args order for better readability?